lib/core/Cas.ts

```diff
@@ -38,7 +38,8 @@ export enum FetchResultCode {
   Success = 'success',
```

Comment:
Should this be pulled out into its own file like how RequestError has a code enum and Response has response status codes?

Reply:
Done.